icgc_song_payloads/OCCAMS-GB/WGS/trial_batch/EGAR00001130626_EGAF00000235882.json

```diff
@@ -0,0 +1,64 @@
+{
+  "studyId": "OCCAMS-GB",
+  "analysisType": {
+    "name": "sequencing_experiment"
+  },
+  "info": {
+    "origin": "EGA"
+  },
+  "samples": [
+    {
+      "specimen": {
+        "submitterSpecimenId": "SS6003306",
+        "specimenTissueSource": "Esophagus",
+        "specimenType": "Primary tumour",
+        "tumourNormalDesignation": "Tumour"
+      },
+      "donor": {
+        "gender": "Male",
+        "submitterDonorId": "04feb20b0eef08db14ff7df6ef52bd0f71fbaebc9fbb6099da2dafedd1ddef9a"
+      },
+      "submitterSampleId": "SS6003306",
+      "matchedNormalSubmitterSampleId": "SS6003304",
+      "sampleType": "Total DNA"
+    }
+  ],
+  "experiment": {
+    "experimental_strategy": "WGS",
+    "sequencing_center": "SC",
+    "platform": "ILLUMINA",
+    "platform_model": "Illumina HiSeq 2000",
+    "sequencing_date": null,
+    "submitter_sequencing_experiment_id": "EGAX00001126739"
+  },
+  "files": [
+    {
+      "fileName": "SS6003306.bam",
+      "fileType": "BAM",
+      "dataType": "Submitted Reads",
+      "fileSize": 160619000000,
+      "fileMd5sum": "e15c29d93563343de3b15d3abbb69e36",
+      "info": {
+        "origin": "EGA",
+        "data_category": "Sequencing Reads",
+        "legacyAnalysisId": "EGAR00001130626_EGAF00000235882"
```

Comment:
@edsu7 @lindaxiang The `legacyAnalysisId` field was to help keep track of ICGC25K identifiers when we carried over files to ARGO, but since we're not planning to include this data transfer in ICGC25K (ie. it won't be available for download at https://dcc.icgc.org/repositories) and we don't make the original BAM files available in ARGO, we can remove this whole `info` section.

Reply:
@edsu7 Yes good point! You're right, we should keep this section then.

Reply:
I agree @hknahal . `legacyAnalysisId` is a reserved field to keep track of ICGC25k identifiers. 
@edsu7  has a good thought regarding this. How about we come up with another field, like `egaIds` to keep EGA related IDs?

Reply:
@edsu7 this is a good idea. It would provide us capability to cross-ref those EGA Ids in future.